includes/functions/functions_lookups.php

```diff
@@ -336,7 +336,7 @@ function zen_requires_attribute_selection($products_id) {
               left join " . TABLE_PRODUCTS_OPTIONS . " po on pa.options_id = po.products_options_id
               where pa.products_id = " . (int)$products_id . "
               and po.language_id = " . (int)$_SESSION['languages_id'] . "
-              group by pa.options_id, pa.products_attributes_id";
+              group by pa.options_id";
```

Comment:
Okay. I need to understand how you arrived at the conclusion that this `group by` was causing a problem.

... cuz with MySQL 5.7 (just tested on 5.7.13) without it an error occurs: 

> Expression #1 of SELECT list is not in GROUP BY clause and contains nonaggregated column `pa.products_attributes_id` which is not functionally dependent on columns in `GROUP BY `clause; this is incompatible with sql_mode=`only_full_group_by`